libs/shared/ui/src/lib/components/header/header.tsx

```diff
@@ -7,24 +7,27 @@ import Truncate from '../truncate/truncate'
 export interface HeaderProps {
   title?: string
   icon?: IconEnum | string
-  iconClassName?: string
   buttons?: ReactNode
   copyTitle?: boolean
   copyContent?: string
   actions?: ReactNode
 }
 
 export function Header(props: HeaderProps) {
-  const { title, icon, buttons, actions, iconClassName = '' } = props
```

Comment:
We don't need to remove the `iconClassName` because the Environment header isn't good right now
<img width="1790" alt="Capture d’écran 2023-09-26 à 15 11 49" src="https://github.com/Qovery/console/assets/533928/39143448-883e-441d-a69c-33b5c930382a">

Reply:
https://github.com/Qovery/console/pull/850/commits/04ae909c3c22114b6676b0d731aa055d813dec56